Week2/git_push.py

```diff
@@ -0,0 +1,37 @@
+import json
+import requests
+
+def create_pull_request(project_name, repo_name, title, description, head_branch, base_branch, git_token):
+    """Creates the pull request for the head_branch against the base_branch"""
+    git_pulls_api = "https://github.com/api/v3/repos/{0}/{1}/pulls".format(
```

Comment:
in python3, you could use
`f"https://github.com/api/v3/repos/{project_name}/{repo_name}/pulls"`